presto-docs/src/main/sphinx/release/release-0.218.rst

```diff
@@ -0,0 +1,43 @@
+=============
+Release 0.218
+=============
+
+General Changes
+---------------
+
+* Add support for using binary (smile) encoding for coordinator-to-worker communication.
+This feature is experimental, and it can be enabled with the ``experimental.internal-communication.binary-transport-enabled`` config option.
+Enabling this feature should help with coordinator scalability, and should reduce network, cpu, and memory usage on the coordinator.
+* Add ST_Area for SphericalGeography
+* Add a new system table ``system.metadata.analyze_properties`` to display available
+properties that can be used in ``ANALYZE`` statement.
+* Optimize performance for queries with COALESCE join keys on top of FULL OUTER JOIN
+* Fix failures in regular expression functions for certain inputs when the pattern contained word boundaries.
+* Consider estimated query peak memory when making cost based decisions.
+* Add an optional additional argument to split_to_map to resolve key conflicts
+* Improve performance for queries with an outer join followed by `LIMIT`.
+* Fix a ``ClassLoader`` crashing issue for plugins providing ``EventListenerFactory``.
+* Add support for role management (see :doc:`/sql/create-role`).  Note, using :doc:`/sql/set-role`
+  requires an up-to-date client library. (:issue:`90`)
+
+
+Security Changes
+----------------
+
+* Make principal hostname configurable in Kerberos authenticator.
+
+
+Hive Connector Changes
+----------------------
+
+* Add support for skipping Glacier files in PrestoS3FileSystem with the ``hive.s3.skip-glacier-objects`` config option, which is false by default.
+* Support Parquet files written with Parquet 1.9+ that use ``DELTA_BINARY_PACKED`` encoding with the Parquet ``INT64`` type.
+* Support dictionary filtering for Parquet v2 files using `RLE_DICTIONARY` encoding.
+* Add option to preload all splits with ``hive.preload-splits-for-grouped-execution`` config option, which is false by default. Turning this on might slightly increase query latency for grouped execution.
```

Comment:
- `might` -> `can`
-  Should we say: `the query latency with grouped execution`